pages/index.tsx

```diff
@@ -1,13 +1,22 @@
-import type { NextPage } from "next";
+import type { GetServerSidePropsContext } from "next";
 import Head from "next/head";
-import Image from "next/image";
 import styles from "../styles/Home.module.css";
 import Announcement from "../components/Announcement";
 import ScheduleWidget from "../components/ScheduleWidget";
 import BellSchedule from "../components/BellSchedule";
 import WeeklySchedule from "../components/WeeklySchedule";
+import { ReceivedAnnouncement } from "../types/db_types";
+import getServerUrl from "../lib/getServerUrl";
+import { InferGetServerSidePropsType } from "next";
+import { Key } from "react";
 
-const Home: NextPage = () => {
+const Home = (
+	props: InferGetServerSidePropsType<typeof getServerSideProps>
+) => {
+	const parseDate = function (d: string) {
+		return new Date(d).toLocaleString().split(",")[0];
+	};
+	console.log("Server url", props.serverUrl);
```

Comment:
Perhaps this debug log is no longer necessary

Reply:
Let’s keep it in.
It’s going to be helpful with all of the vercel preview branches.